pkg/iac-providers/cft/v1/load-file.go

```diff
@@ -40,44 +42,96 @@ func (a *CFTV1) LoadIacFile(absFilePath string) (allResourcesConfig output.AllRe
 	}
 
 	// parse the file as cloudformation.Template
-	fileExt := a.getFileType(absFilePath, &fileData)
-	var template *cloudformation.Template
+	configs, err := a.getConfig(absFilePath, &fileData, nil)
+	if err != nil {
+		zap.S().Debug("unable to normalize data", zap.Error(err), zap.String("file", absFilePath))
+		return allResourcesConfig, err
+	}
+
+	// fill AllResourceConfigs
+	allResourcesConfig = make(map[string][]output.ResourceConfig)
+	var config *output.ResourceConfig
+	for _, resource := range configs {
+		config = &resource
+		config.Line = 1
+		config.Source = a.getSourceRelativePath(absFilePath)
```

Comment:
should the `source` for the nested stack resources be the `template url` or the filepath?
@cesar-rodriguez @kanchwala-yusuf WDYT?

Reply:
I believe, it should be the `template url//filepath`. This way the user can clearly identify the module and the file in which the resource has been initialized

Reply:
I agree with @kanchwala-yusuf